tests/systems/systems_test_data.py

```diff
@@ -308,11 +308,12 @@ def network_factory(*args: Any, **kwargs: Any) -> Any:
         use_next_extras=False,
         sample_batch_size=5,
         nodes_on_gpu=[],
-        is_test=True,
         checkpoint_minute_interval=3 / 60,
         trainer_parameter_update_period=1,
         evaluation_interval={"executor_steps": 10},
         evaluation_duration={"evaluator_episodes": 2},
         checkpoint_best_perf=True,
+        termination_condition={"executor_steps": 100},
```

Comment:
Perhaps we can extends this a bit to something like 30000? 100 seems quite little especially since we were previously doing 5 trainer steps, but I am not 100% how many executor steps would match that.

Reply:
I will give it a try rhe issue is that the coverage bot is complaining

Reply:
I changed the steps to be 5000 because if I change it to 30000 it may take very long time while it is just integration test